ui/tests/e2e/test.js

```diff
@@ -115,9 +115,11 @@ describe("End-to-end", () => {
     // Click on the drop down
     let initial_select = await page.$x("//div[text()='Select...']");
     initial_select[0].click();
-    // Click on the 'Avuncular' chip
-    await page.waitForXPath("//div[contains(text(), 'Avuncular')]");
-    let avuncular = await page.$x("//div[contains(text(), 'Avuncular')]");
+    // Click on the 'Avuncular' option
+    await page.waitForXPath("//span[contains(normalize-space(), 'Avuncular')]");
+    let avuncular = await page.$x(
+      "//span[contains(normalize-space(), 'Avuncular')]"
```

Comment:
I looked into why `"//span[contains(normalize-space(), 'Avuncular')]"` works and `$x("//span[contains(text(), 'Avuncular')]")` doesn't. This [SO post](https://stackoverflow.com/questions/3655549/xpath-containstext-some-string-doesnt-work-when-used-with-node-with-more) explains why.

For future reference, this should be `//span[text()[contains(.,'Avuncular')]]`.